db/migrate/20180412124505_add_priority_score_to_discovery_rules.rb

```diff
@@ -0,0 +1,23 @@
+ class AddPriorityScoreToDiscoveryRules < ActiveRecord::Migration[4.2]
+  class DiscoveryRules < ActiveRecord::Base
+  end
+
+  def up
+    score = DiscoveryRules.distinct.order("priority asc").pluck(:priority).first
```

Comment:
This is the query how we search discovery rules at the moment, can you do the very same query so we know for sure there are no regressions caused by incorrect oder after migration?

    DiscoveryRule.where(:enabled => true).reorder(:priority, :created_at)

Add `unscoped` as well to make sure we get them all. Also use batch processing, so I'd expect something like:

    DiscoveryRule.unscoped.where(:enabled => true).reorder(:priority, :created_at).find_each